app/scripts/util.js

```diff
@@ -85,10 +85,18 @@ var util = {
 
 		return result + ' ' + util.padTime(date.getUTCHours()) + ':' + util.padTime(date.getUTCMinutes()) + ' UTC';
 	},
-	languages: [
-		'java',
-		'py',
-		'c',
-		'js'
-	]
+	clickInsideElement: function(e, elName){
```

Comment:
why did u add this here if you use it just in on file. If you want it to be reusable pls do a behavior.